app/src/main/java/ch/epfl/sdp/musiconnect/events/EventPage.java

```diff
@@ -59,7 +81,7 @@ private void retrieveEventInfo() {
         createDummyEvent(String.valueOf(eid));

```

Comment:
Look at my PR : https://github.com/ayman-mezghani/MusiConnect/pull/143/files 
I set the dummyEvent in the mockdatabase, hence there is no code relative to the tests in the application code